docs/getting-started/install-configure.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Install & Configure
```

Comment:
Much of this document does not make as much sense with all of the provider guides preceding it. I'm not sure what our flow is now for a quick start. Are we expecting that folks install providers, provision infrastructure, delete infrastructure, uninstall providers, then install configurations? Or perhaps more likely, install providers, provision infrastructure, install configuration, provision more infrastructure but using composition? At that point it feels like we are getting a bit beyond a quickstart. The reason for the current structure is that it show off all features of Crossplane with relatively few commands. The danger of starting with just using providers is that users may drop off afterwards and never really get to understanding composition, etc. Counterpoint to that would be that it doesn't matter because they don't understand what is actually happening when they see the quickstart today, but it would be awesome if we could bridge the gap of:
1. keeping the quickstart minimal
2. showing off the full capabilities of Crossplane
3. not overwhelming users with concepts

Right now I feel like we have (1) and (2), and this PR would kind of take us to some version of (2) and (3).

Reply:
This came up today in the community call, but I agree. 

Some of this is in #3398, specifically-

>Quick start vs Reference Implementations
>Quickstarts are specifically designed for people with little or no knowledge of Crossplane. The goal of Quickstart guides is not to show the full power of Crossplane but just to demonstrate how Crossplane connects a Kubernetes cluster to external resources.
>
>Reference Implementations are designed to be a blueprint for "how to do it right". This should be considered a (mostly) comprehensive reference for users that that have experience with Crossplane and are looking for practical examples on how to implement more advanced Crossplane features.

In the Oct 20 Community Call we discussed different goals for what you're describing based on personas:
**1.) A user completely new to Crossplane. May have limited Kubernetes knowledge.** 
This is the target persona of a provider quickstart. How do you demonstrate the core concept of Crossplane communicating with an external API endpoint, entirely through Kubernetes. The quickstart (should) also demonstrates a validation of security credentials between Crossplane and the Provider.

**2.) A user who is familiar with Crossplane or self-service infrastructure APIs** 
This persona is more interested in a more comprehensive example and "magic moment". The discussion today was around creating a new "Composition and Claim" quickstart delivered in a simple fashion (TBD, likely a config package). The user would then apply a small number of configurations or changes to finish the deployment and consume the resources. The main thing here is fast forwarding through all of the lower-level configurations and requirements (e.g., Provider) to focus on full value of Crossplane.

**3.) An end-to-end example using one or more providers**
This would be the "Reference Implementations' mentioned in #3398. These would be "production grade" ways to use Crossplane for a given provider (or set of providers). These should be considered "common/best practices" way to do things. 

Regarding the existing "Getting Started" examples I believe they should be pulled into the Concepts/Components sections to show the configuration and linkage of resources. It's a bit hard to explain this without just building the content

Reply:
Let me know if there are any other questions or comments on this one.

Reply:
@plumbis how would you feel about starting with the managed resources (as you have proposed), but making the quickstart go all the way through composition and configuration packages as it does today? Would that be too long? Perhaps if there were clearly denoted drop-off points it would be tenable?

Reply:
I want that as the "call to action" at the end of these guides. Now that the user has connected Crossplane to their cloud provider and shown it works, the next step is to really highlight the value of Crossplane. 

Maybe this means extending the single guide or maybe it means a second guide, I think that'll become clear once the writing starts. My gut says it's a second "Getting started with Compositions" type guide.

Reply:
Gotcha 👍🏻 I'm not sure I agree, but I defer to your expertise here. I do feel a bit uneasy about merging these guides as-is, which would effectively put our docs into a "broken" state (granted that they wouldn't be showing up as the default). Do you think there is potentially some middle ground where all of the linked issue does not have to be resolved in this PR, but we could be in a state where we would feel good about folks using the `master` docs in the interim?

Reply:
@plumbis so mark all existing quickstart docs as draft and enable the new ones right? Is motivation for getting this PR in prior to other docs so that folks can see where the docs are headed and provide feedback?

Reply:
The specifics I think depend on how the maintainers like yourself prefer to work:

Option 1: **Drafts**
We mark these as drafts. They aren't published on the site but live in the repo. We have this PR finalized and start working on the next set of things. Each set is a "stand alone" topic/set of pages, all marked as drafts until we're ready to flip the switch on everything with one final "release" PR. This PR would fit into this model.

Option 2: **Monolithic PR**
Get it all together, written and ready for publication. The single PR could be 10s of pages of content and anywhere between 50 and 100+ commits. A single large PR is submitted for review containing everything. This PR would likely be closed without committing and re-submitted with the larger collection of content being discussed.

Either option works for me, so I'll leave it up to you on what you'd prefer.

Reply:
This has two challenges:

1. It's slow. Since all docs are loosely coupled it's hard to start on patch-2 until patch-1 is approved. 
2. It will likely create large number of merge conflicts and will require a force push when the branch is merged.

I don't have opinions on (2) but I want to make sure I'm in line with how the maintainers expect code contributions.

Reply:
> This has two challenges.

I presume you're referring to the option of using a new docs branch? If so I'm happy to fall back to a monolithic PR. If I try to think about these changes as I would code we'd aim for incremental changes if possible, but if not fall back to a large PR. I don't think we have any precedent for merging draft anything (code, docs, etc) into master.